test/quantization/fx/test_equalize_fx.py

```diff
@@ -643,14 +644,14 @@ def test_input_weight_equalization_graphs(self):
         linearRelu_node_list = [
             ns.call_function(torch.mul),
             ns.call_function(torch.quantize_per_tensor),
-            ns.call_module(nniq.LinearReLU),
+            ns.call_module(nni.modules.fused.LinearReLU),
```

Comment:
this is the same as: nni.LinearReLU

Reply:
actually these should be lowered I feel, are we using reference models here?

Reply:
As discussed offline, these changes will be reverted